gis/assets/traffic_signals/README.md

```diff
@@ -0,0 +1,104 @@
+﻿# Traffic Signals dataset
+
+This is one of the datasets managed by the Traffic Control group (https://github.com/CityofToronto/bdit_vz_programs#datasets-and-their-owners).  
+
+Data is updated by automatic feed in Traffic Control and stored in their Oracle database. Here, we set up an airflow process to automatically extract Traffic Signal data from the Oracle database using Foreign Data Wrappers and update the table in the bigdata RDS. The pipeline consists of the following steps:  
```

Comment:
This should describe the traffic signals data set and also explain what's being pulled in the DAG.